authentication/auth.go

```diff
@@ -558,6 +556,32 @@ func GenerateTokenFromDefaultsV2(schemaURL string, accountServiceURL string, url
 	return token, ""
 }
 
+func addAdditionalMetadata(surveyMetadata map[string]interface{}, requiredSchemaMetadata []Metadata) []Metadata {
+	for _, metadataVal := range surveyMetadata {
+		for key, value := range metadataVal.(map[string]interface{}) {
+			if isExistingRequiredSchemaMetadata(requiredSchemaMetadata, key) {
+				continue
+			} else {
+				newMetadata := Metadata{}
+				if strings.Contains(value.(string), "True") || strings.Contains(value.(string), "False") {
```

Comment:
How come we're expecting the URL param to be `True` or `False` when the value is `true/false` - should we handle both in URL?

Reply:
My reasoning for `True` and `False` was to follow the Python convention for `True/False` with capital letters, so if someone wanted to have a parameter set to a boolean they would follow this standard. If we handle both, there's still the chance that someone could intend for 'true' or 'false' to be a string rather than a boolean.

Reply:
The standard is to support all toggle words, `on`, `off` etc. But here I would say handle it without case-sensitive, so compare the equal case so that it handled both upper and lower case.

Reply:
No problem, will handle in the new PR